### PR TITLE
Add assignment pdfs.

### PR DESCRIPTION
**Pull request by Javid912**
I have added the assignment.md file, I added the images from the notes from our first meeting as @fpopic sugested

### CODE REVIEW

File: airflow/dags/dag0002.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 }
 
 dag = DAG(
-    'gcs_file_check_dag_2',
+    dag_id= 'gcs_file_check_dag',
```
Comment (fpopic) [inline airflow/dags/dag0002.py:17]:
```suggestion
    dag_id='gcs_file_check_dag',
```

**Comment by fpopic** [comment]
Hi @Javid912, 

- you have to go to your local `main` branch, 
- pull latest changes from `remote origin main` to your local `main` branch, 
- change to branch` assignment`
- do either `git rebase main` or `git merge main` and resolve conflicts 
  - either manually hard core with VIM
  - using IntelliJ https://www.youtube.com/watch?v=ZdOufSFNHnM
  - or vs code https://www.youtube.com/watch?time_continue=257&v=HosPml1qkrg&embeds_referring_euri=https%3A%2F%2Fwww.google.com%2Fsearch%3Fq%3Dvs%2Bcode%2Bresolve%2Bmerge%2Bconflicts%2Bgit%2Blens%26sca_esv%3D599255904%26rlz%3D1C5CHFA_enDE1007DE1008%26sxsrf%3DACQV&source_ve_path=MjM4NTE&feature=emb_title

**Comment by fpopic** [comment]
did it work?

**Comment by Javid912** [comment]
I did all that, I think it worked to resolve conflicts on the assignment branch. can you review the changes so we can merg the assignment branch?
I also tried to work on the dag and read the logs to find out why it kept failing..
a review of the whole DAG writing  module is also in progress

File: airflow/dags/dag_assignment.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 }
 
 dag = DAG(
-    'gcs_file_check_dag_2',
+    dag_id= 'gcs_file_check_dag',
```
Comment (fpopic) [inline airflow/dags/dag_assignment.py:17]:
```suggestion
    dag_id='gcs_file_check_dag',
```

**Review by fpopic — APPROVED** [review]

**Pull request merged**